### ISSUE TITLE
Bring in data from status.modern.ie

### ISSUE THREAD

**Issue by ebidel** [GoogleChrome/chromium-dashboard#114]
http://status.modern.ie/features

So we don't have to manually keep IE data up to date.

![screen shot 2014-04-10 at 10 33 47 am](https://cloud.githubusercontent.com/assets/238208/2671083/4bb7551c-c0d6-11e3-962a-89afc90c814d.png)

Modern status has also mixed in the webkit data: https://github.com/MicrosoftEdge/Status/pull/226/files

**Comment by phistuck** [comment]
We should not use the WebKit status data. See https://github.com/MicrosoftEdge/Status/pull/226#issuecomment-97672727 for details.

**Comment by RByers** [comment]
Let's just focus on the Edge data here, filed #288 to discuss WebKit vs. Safari.

**Comment by phistuck** [comment]
Regarding Edge, priorities should also be considered here. "Under consideration" and "High" equals "Public support". I think "Medium" also counts as "Public support" (less strong, though). "Low" is probably "Mixed public signals" (same as without a priority).

@jacobrossi - please, correct me if I am wrong.

**Comment by jacobrossi** [comment]
Here are the specific definitions we use:

Roadmap Priority: Low — We are still evaluating this technology. There may be significant spec stabilization, foundational work, or additional community input required before we can begin development.
Roadmap Priority: Medium — Development is likely for a future release.
Roadmap Priority: High — We intend to begin development soon.

You can certainly map Med/High to "Public Support" as we're making a positive signal here that we would like to implement the feature. 

IMO  "mixed public signals", "public skepticism", and "no public signals" don't really accurately describe what we're conveying with Low priority.  We're evaluating the technology and haven't made a determination as to whether we support it or not.  In some cases, this is because we need more data (for example, having an engineer investigate in the feasibility in our codebase, gathering compat data on the web, or talking to developers or framework authors for their input). In others, it's simply because we consider the scenario lower priority and so we haven't spent much time yet in establishing a definitive opinion.  In others, it's because we're waiting for outstanding issues to be resolved in standards.

I think an under investigation/consideration status would feel more accurate for these cases.

**Comment by RByers** [comment]
@jacobrossi we're starting to explore how best to use your data (some discussion also in #303).  I've noticed the metadata linking your issues to our issues (or our bugs) isn't present in your JSON feed.  Is that intentional?  I think we'd like to start by trying to identify all matching features across the two data sources and figure out how our status values would change if we updated them based on your data.

**Comment by phistuck** [comment]
@RByers - the Edge Status JSON contains Chrome dashboard IDs and matches and imports data from the Chrome dashboard JSON. If you load Edge Status, you can see it loading the Chrome JSON.

**Comment by phistuck** [comment]
@RByers - example - https://github.com/MicrosoftEdge/Status/blob/5ed4471ae311983ca61b2202afac19e90f2ad93b/status.json#L18

**Comment by RByers** [comment]
Oh cool, thanks @phistuck - I didn't realize that generic `id` might match ours.  @mdittmer sounds to me like this is another good place to start - just replace our Edge status URL with an official Edge platform status link for every feature which already links back to us.

**Comment by jacobrossi** [comment]
FWIW if anyone is interested in taking this as an opportunity to simplify / clean up the JSON schema, I'd be game.  It is what it is today mostly because that's how it was in the Chromium Dashboard.  Things like "ie-status" are warts because it refers to Edge now. Also many of the values for Chromium are store in a different format than for other browsers (e.g. impl_status_chrome vs safari_views { }, "enabled by default" vs "shipped", etc.). We have a ton of code that normalizes things in this regard.

Once we fetch the Chromium data and merge it with our data, we end up with an in-memory object that looks something like:

````
{
    id:
    name:
    summary:
    category:
    spec: {
        name:
        shortname:
        link:
        organization:
        status:
    }
    docs: {
        msdn:
        mdn: 
    }
    browsers: {
        chrome: {
            status:
            link:
            prefixOrFlagRequired:
            version:
        }
        edge: {
            status:
            priority:
            uservoice:
            demo:
            prefixOrFlagRequired:
            version:
        }
        firefox: {
            status:
            link:
            prefixOrFlagRequired:
            version:
        }
        safari: {
            status:
            link:
            prefixOrFlagRequired:
            version:
        }
        opera: {
            status:
            link:
            prefixOrFlagRequired:
            version:
        }
    }
}
````

Not all of this information is available across browsers (but could be gathered). We also simplify the non-Edge browser statuses to: Shipped, Deprecated, In Development, or Not Supported. For Edge we use Shipped, Deprecated, In Development, Under Consideration (optionally, with priority level), Not Currently Planned.

**Comment by ebidel** [comment]
@jacobrossi great idea. Filed https://github.com/GoogleChrome/chromium-dashboard/issues/321 for discussion

**Comment by RByers** [comment]
Oh, I didn't event realize that Edge's JSON schema came from chromestatus - that explains a lot ;-).

**Comment by jpmedley** [comment]
Because Edge has been based on Chromium for several years, there's now no data to import.
